app/components/Player/Player.tsx

```
@@ -92,7 +92,7 @@ export const PlayerComponent: React.FC<PlayerComponentProps> = ({
         const conProps = {
           ...(asset?.asset?.playbackPolicy && {
             accessKey: generateAccessKey(
-              activeAccount?.address!,
+              user?.address!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)